loafer/ext/aws/providers.py

```diff
@@ -47,7 +47,10 @@ async def fetch_messages(self):
 
     async def _client_stop(self):
         async with self.get_client() as client:
-            await client.close()
+            try:
+                await client.close()
+            except RuntimeError:
+                raise ProviderRuntimeError()
```

Comment:
Acho que poderia encadear a exception `ProviderRuntimeError ` ao `RuntimeError`, o que acha?

ficaria assim:

```suggestion
            except RuntimeError as exc:
                raise ProviderRuntimeError() from exc
```

Dessa forma, quando levantar um error `ProviderRuntimeError` ele encadeia a referencia da exception ao `RuntimeError` que é de fato onde a exception foi levantada.

Reply:
boa, feito https://github.com/olist/olist-loafer/pull/17/commits/59c15b0cd822983f6ac208fed798e49bfadc8b50